tests/unit/help_test.sh

```diff
@@ -3,6 +3,51 @@
 include './tests/unit/utils.sh'
 include './src/help.sh'
 
+function oneTimeSetUp()
+{
+  declare -g PATH_TO_GIT_REPOSITORY
+  declare -g PATH_TO_KW_REPOSITORY
+
+  PATH_TO_GIT_REPOSITORY="${SHUNIT_TMPDIR}/git_repo"
+  PATH_TO_KW_REPOSITORY=$(realpath "$(dirname "${BASH_SOURCE[0]}")"/../..)
+
+  setupGitRepository
+}
+
+function oneTimeTearDown()
+{
+  teardownGitRepository
+}
+
+# Execute a git command in our temporary git directory.
+function git_cmd()
+{
+  git -C "${PATH_TO_GIT_REPOSITORY}" "$@"
+}
+
+# We may want supress git's output so it doesn't show up when running tests.
+function git_cmd_silent()
+{
+  git_cmd "$@" > /dev/null 2>&1
```

Comment:
Git already has the `--quiet` option, which is better than using redirect to `/dev/null`.